### ISSUE TITLE
ProjectSpoon PRO Subscriptions - Easter Sale 50% Off

### ISSUE THREAD

**Issue by angek** [angek/ProjectSpoon#53]
for a limited time, we are offering **50% off** our **6 and 12 month** subscription plans.
Simply enter the promo code **EASTER50** at checkout to receive the discount.

https://projectspoon.net/subscriptions

**Comment by ssnobben** [comment]
HI didn’t find the promo code entrance :) will buy a future subscription but would like to know if this pkg is Joomla compatible too. 

And would nice to have some popup info/tool tips for all icons/buttons for explanations of what they do. Is there a .ini English language file that you can get so you can start translating your non existing language for Projectspoon?

**Comment by angek** [comment]
When you click on the subscription you ate presented with a promo field. Enter the code and continue the checkout. The discount is automatically applied.

Yes there are language files for each component, module and plugin that makes up the ProjectSpoon package.

**Comment by angek** [comment]
As it stands, ProjectSpoon is not currently compatible with Joomla 4. Joomla 4 is still in Alpha release. I will be migrating to Joomla 4 but not until the final Joomla 4 beta release has come out.
